packages/runtime/src/traits/core/style.tsx

```diff
@@ -2,22 +2,25 @@ import { createTrait } from '@sunmao-ui/core';
 import { Static, Type } from '@sinclair/typebox';
 import { TraitImplementation } from '../../types/RuntimeSchema';
 
-const StyleTrait: TraitImplementation<Static<typeof PropsSchema>> = ({
-  styleSlot,
-  style,
-}) => {
+const StyleTrait: TraitImplementation<Static<typeof PropsSchema>> = ({ styles }) => {
+  const customStyle: Record<string, string> = {};
+  styles.forEach(style => {
+    customStyle[style.styleSlot] = style.style;
+  });
   return {
     props: {
-      customStyle: {
-        [styleSlot]: style,
-      },
+      customStyle,
     },
   };
 };
 
 const PropsSchema = Type.Object({
-  styleSlot: Type.String(),
-  style: Type.String(),
+  styles: Type.Array(
```

Comment:
Do any examples need migration?

Reply:
Yes, I have already updated examples.

Reply:
Well, actually there is no style trait example, only some tests.